src/dotnet/commands/dotnet-new/LocalizableStrings.cs

```diff
@@ -0,0 +1,35 @@
+namespace Microsoft.DotNet.Tools.New
+{
+    internal static class LocalizableStrings
+    {
+        public static const string AppName = "dotnet new";
+
```

Comment:
In what context is "dotnet new" localizable. Isn't it what I always have to type?